module/solutions/helper.js

```diff
@@ -2243,7 +2243,7 @@ module.exports = class SolutionsHelper {
   ) {
     return new Promise(async (resolve, reject) => {
       try {
-          let duplicateSolutionDocument = await solutionsUtils.importFromSolution(  
```

Comment:
@Prajwal17Tunerlabs why we changed the variable name? original one was more meaningful right?